pysat/tests/test_utils_files.py

```diff
@@ -21,10 +20,11 @@
 
 class TestParseDelimitedFilenames():
     """Unit tests for the `parse_delimited_filename` function."""
-    temporary_file_list = False
 
     def setup(self):
-        """Runs before every method to create a clean testing setup."""
+        """Set up the unit test environment for each method."""
+
+        self.temporary_file_list = False
```

Comment:
Not using inheritance here, so I moved the definition to method-level setup.  We could define this as a class-level setup if we plan to expand this to repeat tests with `temporary_file_list=True`.